cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -6,13 +6,59 @@
 
    Description:
 
-   Create a hero molecule. See [GHE]/flapjack/Modules-V1/wiki/Heros
+   Create a hero molecule.
+
+   settings:             An object with the following options for settings.
+
+   REQUIRED:
+
+   settings.heading:     A string containing title of module.
+
+   settings.body:        A string containing the content to display below the
+                         heading (typically a paragraph).
+
+   settings.image_url:  A string containing the url of the image to be placed
+                         adjacent to the text
+
+   OPTIONAL:
+
+   settings.background: A string containing the color code (in any format) to
+                         set as the hero background color
+
+   settings.links:       A tuple of items to create a list of links.
+                         Objects include a url and text to display.
+
+                         link_url: A string for the URL of the link.
+                         link_text: A string for the text of the link.
+
+   See [GHE]/flapjack/Modules-V1/wiki/Heros
 
    ========================================================================== #}
 
-{% macro render() %}
-<div>
-  {# TODO: Implement hero macro with actual content. #}
-  HERO
-</div>
+{% macro render( settings ) %}
+
+{% if settings.heading and settings.body and settings.image_url %}
+<section class="hero"
+         {{ 'style=background-color:' +
+             settings.background + ';'
+             if settings.background else '' }}>
+    <div class="wrapper hero_wrapper">
+        <div class="hero_text">
+            <h1 class="hero_heading">{{ settings.heading }}</h1>
+            <p class="hero_subhead">{{ settings.body }}</p>
+            {% if settings.link_text and settings.link_url %}
+            <p class="hero_cta">
+                <a href="{{ settings.link_url }}">
```

Comment:
Yeah, the `#` is just for displaying it in the prototype. This won't be used in events because that will use the Featured Content Module, so I don't think that scenario will ever come up.